.travis.yml

```diff
@@ -27,7 +26,6 @@ script:
   - npm run build:prod
 
 after_success:
-  - npm install -g firebase-tools
   - npm run deploy
   # - npm run codecov
 
```

Comment:
Travis actually added Firebase to the supported deploy stuff, so lets actually give this a try:

```yaml
deploy:
  skip_cleanup: true
  provider: firebase
  token:
    secure: $FIREBASE_TOKEN
    project: $TRAVIS_BRANCH
```

That will also mean we have to add this to `.firebaserc`:
```
"master": "devshare-int",
```